library/ix-dev/community/tiny-media-manager/upgrade_strategy

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/python3
+import json
+import re
+import sys
+
+from catalog_update.upgrade_strategy import semantic_versioning
+
+
+RE_STABLE_VERSION = re.compile(r'\d+\.\d+\.\d+(\.\d+)?')
```

Comment:
So it can be some thing like `1.1.12.13`?

Reply:
Yes, its varying between 3 and 4 "parts"
![image](https://github.com/truenas/charts/assets/47820033/128cda2b-8fe1-4880-bb4b-7c7b143ed320)